src/prted/pmix/pmix_server_dyn.c

```diff
@@ -549,9 +551,11 @@ static void interim(int sd, short args, void *cbdata)
             pmix_server_cache_job_info(jdata, info);
 
         } else if (PMIX_CHECK_KEY(info, PMIX_DEBUG_STOP_IN_APP)) {
+            rank = PMIX_RANK_WILDCARD;
             prte_set_attribute(&jdata->attributes, PRTE_JOB_STOP_IN_APP, PRTE_ATTR_GLOBAL,
-                               info->value.data.string, PMIX_STRING);
```

Comment:
I'm not sure this is correct - I'll have to look in the code, but we need to have that label somewhere so we know where to tell the app to stop.